packages/react-server-dom-esm/src/client/ReactFlightDOMClientBrowser.js

```diff
@@ -78,14 +78,14 @@ function startReadingFromStream(
     value,
   }: {
     done: boolean,
-    value: ?any,
+    value: ?Uint8Array,
     ...
   }): void | Promise<void> {
-    if (done) {
+    if (done || value == null) {
```

Comment:
That shouldn't happen. It should just error if `value` is `null` here. But not worth the extra runtime code. The previous type-casting below (`const buffer: Uint8Array = (value: any);`) could've just stayed

Reply:
I guess I thought that done would always return true when value was null. I think you were right!